html_sanitizer/sanitizer.py

```diff
@@ -334,11 +334,16 @@ def sanitize(self, html):
         elif isinstance(self.autolink, dict):
             lxml.html.clean.autolink(doc, **self.autolink)
 
+        all_allowed_attrs = {
+            attr for attrs in self.attributes.values() for attr in attrs
+        }
```

Comment:
I needed some time to digest this :-) reading list comprehensions with more than one for...in is hard.

This might be clearer? `all_allowed_attrs = set(itertools.chain.from_iterable(self.attributes.values()))`. Or maybe even assemble he set of allowed attributes using a simple for loop: `for attrs in self.attributes.values(): all_allowed_attrs |= attrs`

On the other hand do we even need the `safe_attrs_only` with our very restrictive list of allowed attributes? Might it not be easier to just drop the `safe_attrs_only=True` call?

Reply:
It does look like there's no point in keeping `safe_attrs_only=True`, should I had a commit to remove it and the `all_allowed_attrs` stuff?
Otherwise if kept, I would prefer the simple for loop since I'm not very familliar with `itertools`.

Reply:
I think removing `safe_attrs_only=True` is the better way. The safe attributes list is not necessary for our use case. If someone provides e.g. `onclick` as one of the allowed attributes they should get what they ask for...